deploy/.env.template

```
@@ -0,0 +1,12 @@
+# deploy/.env.example
+
+# JWT 설정
+JWT_SECRET=
+JWT_ACCESS_EXPIRATION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ACCESS_EXPIRATION key should go before the JWT_SECRET key

(UnorderedKey)